acoular/environments.py

```diff
@@ -382,9 +385,12 @@ class OpenJet(FlowField):
     #: Diameter of the nozzle, defaults to 0.2 .
     D = Float(0.2, desc='nozzle diameter')
 
+    #: Nondimensional length of zone of flow establishment (jet core length), defaults to 6.2
+    l = Float(6.2, desc='flow establishment length')
```

Comment:
What is the reason for the different default values between `SlotJet` and `OpenJet`

Reply:
This is indeed different for slot jet and round jet according to Albertson1950.